features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SystemRequirementConfiguration.kt

```diff
@@ -0,0 +1,122 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay
+
+import com.datadog.android.api.InternalLogger
+import com.datadog.android.api.feature.FeatureSdkCore
+import com.datadog.android.core.InternalSdkCore
+import com.datadog.android.sessionreplay.internal.prerequisite.CpuRequirementChecker
+import com.datadog.android.sessionreplay.internal.prerequisite.DefaultSystemRequirement
+import com.datadog.android.sessionreplay.internal.prerequisite.MemoryRequirementChecker
+import com.datadog.android.sessionreplay.internal.prerequisite.SystemRequirement
+import java.util.concurrent.Executors
+
+/**
+ * This class defines the system requirements necessary to enable the Session Replay feature.
+ *
+ * @param minCPUCoreNumber The minimum CPU core number.
+ * @param minRAMSize The minimum RAM in megabytes.
+ */
+class SystemRequirementConfiguration internal constructor(
+    val minCPUCoreNumber: Int,
+    val minRAMSize: Int,
```

Comment:
We could rename to minRAMSizeMb to make it clearer. WDYT?

Reply:
good idea, fixed